tasks/main.yml

```diff
@@ -1,32 +1,55 @@
 ---
-- name: Ensure /root/.aws/ is there
+- name: Update user directory
+  set_fact:
+    aws_cwa_user_path: "/home/{{ aws_cwa_local_user }}"
+  when:
+    - aws_cwa_local_user != "root"
+    - aws_cwa_local_user == "/" # When not default, assume user has overridden.
```

Comment:
Use `and` or `or` plus is that second condition correct? should it be `aws_cwa_user_path` ?

Reply:
This is wrong but I thought I fixed this already. Maybe I forgot to push???

Reply:
Yes, I did... Sorry.